airrohr-firmware/airrohr-firmware.ino

```diff
@@ -405,6 +407,181 @@ SoftwareSerial serialGPS(GPS_SERIAL_RX, GPS_SERIAL_TX, false, 512);
 #define serialGPS (Serial2)
 #endif
 
+/*****************************************************************
+ * MHZ19 declarations                                           *
+ *****************************************************************/
+class CMHZ19Sensor
+{
+public:
+  CMHZ19Sensor(Stream& serialPort);
```

Comment:
This appears to be two space indentation. The rest is single tab indentation. While I personally hate tab indentation with a passion, I think it would be better to be consistent here.